MSW/Faker - arrays of enum values are not generating properly

<!-- Thanks for reporting an issue! Please fill out the blanks below. -->

## What are the steps to reproduce this issue?

I was able to get a minimal repro stood up:
https://stackblitz.com/edit/vitejs-vite-xqak8w?file=src%2Fhandlers%2Fexample.ts&view=editor

## What happens?

It looks like for some reason the fake type is being generated as an array of array of enums instead of just the array of enums that it should be.

## What were you expecting to happen?

…

## Any logs, error output, etc?

Here's the typescript error that's also visible in the IDE

```sh
 ❯ npm run build

> vite-react-typescript-starter@0.0.0 build
> tsc -b && vite build

src/handlers/example.ts:36:3 - error TS2322: Type '(2 | 1 | 3)[][] | undefined' is not assignable to type 'ExampleEnum[] | null | undefined'.
  Type '(2 | 1 | 3)[][]' is not assignable to type 'ExampleEnum[]'.
    Type '(2 | 1 | 3)[]' is not assignable to type 'ExampleEnum'.

36   exampleProperty: faker.helpers.arrayElement([
     ~~~~~~~~~~~~~~~

  src/handlers/example.ts:22:3
    22   exampleProperty?: ExampleEnum[] | null;
         ~~~~~~~~~~~~~~~
    The expected type comes from property 'exampleProperty' which is declared here on type 'ExampleResponse'


Found 1 error.
```


## Any other comments?

…

## What versions are you using?

```
  System:
    OS: Linux 5.0 undefined
    CPU: (8) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 0 Bytes / 0 Bytes
    Shell: 1.0 - /bin/jsh
  npmPackages:
    @tanstack/react-query: ^5.56.2 => 5.56.2 
    axios: ^1.7.7 => 1.7.7 
    msw: ^2.4.9 => 2.4.9 
    orval: ^7.1.1 => 7.1.1 
    react: ^18.3.1 => 18.3.1
```

--- a/packages/mock/src/faker/getters/scalar.ts
+++ b/packages/mock/src/faker/getters/scalar.ts
@@ -162,6 +162,7 @@ export const getMockScalar = ({
       }
       return {
         value,
+        enums: item.enum,
         imports: numberImports,
         name: item.name,
       };

--- a/tests/specifications/enums.yaml
+++ b/tests/specifications/enums.yaml
@@ -114,6 +114,8 @@ components:
     Dog:
       type: object
       properties:
+        group:
+          $ref: '#/components/schemas/DogGroup'
         petsRequested:
           type: array
           items:
@@ -132,3 +134,10 @@ components:
           type: array
           items:
             type: string
+    DogGroup:
+      enum:
+        - 1
+        - 2
+        - 3
+      type: integer
+      format: int32

